cg/apps/tb/models.py

```diff
@@ -27,6 +27,7 @@ class TrailblazerAnalysis(BaseModel):
     progress: Optional[float] = 0.0
     data_analysis: Optional[str]
     ticket: Optional[str]
+    uploaded_at: Optional[str]
```

Comment:
str or datetime?

Reply:
I initially thought if DateTime but then I saw that the other model attributes where str, even date related attributes like started_at. I think we either keep it as is or update the data types for all the attributes. What do you recon?

![Screenshot 2022-12-01 at 11 57 21](https://user-images.githubusercontent.com/107848458/205035571-36e6be2f-fd67-4644-bd44-72621f1f93a1.png)

Reply:
I believe that it is str because datetime can not be jsonified when sent through the API, so that is why most of the attributes are str

Reply:
So it would be good to have it as a datetime (since that it is what it represent) in all places and then make a conversion  to str when calling the json function